scalyr_agent/tests/scalyr_logging_test.py

```diff
@@ -371,6 +371,20 @@ def test_force_valid_metric_or_field_name(self):
         fixed_name2 = scalyr_logging.AgentLogger.force_valid_metric_or_field_name(name2)
         self.assertEqual(fixed_name2, name2)
 
+    def test_output_to_file_with_stdout(self):
+        self.__logger.info("Hello world", force_stdout=True)
```

Comment:
It would be good to assert that the message is also output to stdout (in addition to the file).

For that, we will likely need to patch sys.stdout and sys.stderr.